sol-did/programs/sol-did/src/instructions/update.rs

```diff
@@ -13,7 +13,7 @@ pub fn update(
         data.nonce += 1;
     }
 
-    data.set_services(update_arg.services)?;
+    data.set_services(update_arg.services, false)?;
```

Comment:
Is there a reason allow_overwrite is set to false with no option for the user to pass it in as true here? In DidSolService the default is false but allows it to be passed in. Will this just not be accessible by the user?

Reply:
It's only accessible to the user via add_service.

the function signature of `set_services` is not allow_overwrite, but allow_duplicates (that then are filtered). In `addService` duplicates are allowed, because an overwrite Service will be duplicate to the existing service. The old service will be filtered out.

For the `update` transaction the user passes A LIST OF Services. Here, no duplicates are allowed. Therefore `false`

Reply:
Ah okay great, I understand... Thanks!